src/store/inputSlice.js

```diff
@@ -146,6 +147,7 @@ function computeDriveInput(state, action) {
   driveInput.steer = driveGamepad["RightStickX"] + getAxisFromKeys(pressedKeys, "ARROWLEFT", "ARROWRIGHT");
   driveInput.tankLeft = driveGamepad["LeftStickY"] + getAxisFromKeys(pressedKeys, "ARROWDOWN", "ARROWLEFT")
   driveInput.tankRight = driveGamepad["RightStickY"] + getAxisFromKeys(pressedKeys, "ARROWRIGHT", "ARROWUP");
+  driveInput.activeSuspension = driveGamepad["LeftStickX"] + getAxisFromKeys(pressedKeys, "B", "M");
```

Comment:
After thinking about it, I'm not sure if we should map it to the left stick, since that's very easy to accidentally move in the X direction while driving with the Y direction. I'd say instead we should use the DPad on the controller. Those are buttons, not axes, so the code would be slightly different. Something like:

```javascript
driveInput.activeSuspension = getAxisFromButtons(driveGamepad, "DPadDown", "DPadUp") + getAxisFromKeys(pressedKeys, "B", "M");
```